.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -50,4 +50,4 @@ Base links to copy
 > Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
 
 ### Link to Definition of Done
-More and detailed information on the *definition of done* can be found [on Confluence](https://docs.hpi-schul-cloud.org/pages/viewpage.action?pageId=92831762)
+More and detailed information on the *definition of done* can be found [on Confluence](https://docs.dbildungscloud.de/x/mAITD)
```

Comment:
It is better to remove this for an Uniform template designs with hpi-schul-cloud/nuxt-client ?